From b8c16e65ce9bc947d4c330e80345f85e331096a4 Mon Sep 17 00:00:00 2001
From: Andrzej <ndrwrdck@gmail.com>
Date: Wed, 25 Nov 2015 23:40:55 +0000
Subject: [PATCH] Fixed crash caused by commit 6431653

---
 panel-plugin/pulseaudio-menu.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/panel-plugin/pulseaudio-menu.c b/panel-plugin/pulseaudio-menu.c
index e055f71..cd6b882 100644
--- a/panel-plugin/pulseaudio-menu.c
+++ b/panel-plugin/pulseaudio-menu.c
@@ -182,6 +182,7 @@ pulseaudio_menu_run_audio_mixer (PulseaudioMenu   *menu,
 
 static void
 pulseaudio_menu_volume_changed (PulseaudioMenu   *menu,
+                                gboolean          should_notify,
                                 PulseaudioVolume *volume)
 {
   g_return_if_fail (IS_PULSEAUDIO_MENU (menu));
@@ -270,7 +271,7 @@ pulseaudio_menu_new (PulseaudioVolume *volume,
   gtk_menu_shell_append (GTK_MENU_SHELL (menu), mi);
   g_signal_connect_swapped (G_OBJECT (mi), "activate", G_CALLBACK (pulseaudio_menu_run_audio_mixer), menu);
 
-  pulseaudio_menu_volume_changed (menu, menu->volume);
+  pulseaudio_menu_volume_changed (menu, FALSE, menu->volume);
 
 
   return GTK_WIDGET (menu);
--
libgit2 0.23.2

